AGGREGATE.md

```diff
@@ -187,8 +189,22 @@ The `payload` will need to contain all the information needed for the aggregatio
 
 The payload should be encrypted via [HPKE](https://datatracker.ietf.org/doc/draft-irtf-cfrg-hpke/), to public keys specified by the processing origins at some well-known address  `/.well-known/aggregation-service/keys.json` that the browser can fetch. Note that we are avoiding using the `attribution-reporting` namespace because many APIs may want to use this infrastructure beyond attribution reporting.
 
-**TODO**: more formally specify `payload`.
+For the MPC protocol, we propose the contents of this base64-encoded encrypted payload will be a [CBOR](https://cbor.io) map with the following structure:
+
+```
+// CBOR
+{
+  "version": "<api version>",
+  "operation": "hierarchical-histogram",  // Allows for the service to support other operations in the future
+  "privacy_budget_key": "<field for server to do privacy budgeting>",
+  "scheduled_report_time": <timestamp in msec>,
+  "reporting_origin": "https://reporter.example",
+  "dpf_key": <binary serialization of the DPF key>,
+}
+```
+Note that the `dpf_key` value would differ in the two payloads.
 
+For the single-server design, the `dpf_key` field would be replaced with `"data": { "bucket": <bucket>, "value": <value> }`. If two `aggregationServices` are specified, one payload (chosen randomly) would contain that data and the other would instead contain `"data": {}`.
```

Comment:
```suggestion
Payloads generated with the `single-server` processingType will directly encode the bucket and value of the histogram contribution instead of a DPF key. The `dpf_key` field will be replaced with `"data": { "bucket": <bucket>, "value": <value> }`. 

If two `aggregationServices` are specified, one payload (chosen randomly) would contain that data and the other would instead contain `"data": {}`.
```